contracts/axelar-gateway/src/testutils.rs

```diff
@@ -58,6 +58,10 @@ pub fn get_rotation_hash(env: &Env, new_signers: WeightedSigners) -> BytesN<32>
         .into()
 }
 
+pub fn get_signers_hash(env: &Env, new_signers: WeightedSigners) -> BytesN<32> {
```

Comment:
define a method on WeightedSigners type instead

Reply:
this is related to the `rotate_signers` event, which will be in a separate PR.